### PR TITLE
KAFKA-5137 : Controlled shutdown timeout message improvement

### PR DESCRIPTION
**Pull request by umesh9794**
This PR improves the warning message by adding correct config details.

### CODE REVIEW

**Comment by cotedm** [comment]
@umesh9794 I believe you can also hit ``request.timeout.ms`` and get an IOException here https://github.com/umesh9794/kafka/blob/f2b35a8e56d83301a42a09fef61a9ba752acce70/core/src/main/scala/kafka/server/KafkaServer.scala#L446

Also there is another place this is getting logged here that could be fixed:
https://github.com/umesh9794/kafka/blob/f2b35a8e56d83301a42a09fef61a9ba752acce70/core/src/main/scala/kafka/server/KafkaServer.scala#L535

@ijuma do you see any other possible socket timeouts that can get hit here?

**Comment by umesh9794** [comment]
Thanks for reviewing it @cotedm. Sorry I missed those possibilities. Will fix them and update the PR.

**Comment by umesh9794** [comment]
@cotedm, @ijuma , I have updated the message. Please review.

**Comment by cotedm** [comment]
I think it's an improvement over the existing message. @ijuma what do you think?

File: core/src/main/scala/kafka/server/KafkaServer.scala [hunk 1]
```diff
@@ -458,7 +458,7 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
             catch {
               case ioe: IOException =>
                 ioException = true
-                warn("Error during controlled shutdown, possibly because leader movement took longer than the configured socket.timeout.ms: %s".format(ioe.getMessage))
+                warn("Error during controlled shutdown, possibly because leader movement took longer than the configured controller.socket.timeout.ms or request.timeout.ms: %s".format(ioe.getMessage))
```
Comment (ijuma) [inline core/src/main/scala/kafka/server/KafkaServer.scala]: Is it worth saying `and/or` to be clearer that it could be either or both?

Comment (umesh9794) [reply core/src/main/scala/kafka/server/KafkaServer.scala]: Thanks for the review @ijuma. Agree, `and/or` makes more sense. Will update the message.

**Review by ijuma — APPROVED** [review]
Thanks, LGTM.

**Pull request closed without merging**